src/tests/Microsoft.PowerFx.Core.Tests/IntellisenseTests/SuggestTest.cs

```diff
@@ -1,7 +1,6 @@
 ﻿// Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-using System.Globalization;
```

Comment:
>Globalization [](http://example.com/codeflow?start=13&length=13)

Can we revert these files? (leftover from a previous iteration; adds a using we don't need)